src/main/java/org/jitsi/jigasi/AudioModeration.java

```diff
@@ -463,6 +456,26 @@ public void mute()
         }
     }
 
+    /**
+     * Raises hand, by adding the extension. We remove the lower hand extension first.
+     */
+    private void raiseHand()
+    {
+        OperationSetJitsiMeetToolsJabber jitsiMeetTools = this.jvbConference.getXmppProvider()
+                .getOperationSet(OperationSetJitsiMeetToolsJabber.class);
+
+        ChatRoom mucRoom = this.jvbConference.getJvbRoom();
+
+        if (mucRoom instanceof ChatRoomJabberImpl)
+        {
+            // remove the default value which is lowering the hand
+            ((ChatRoomJabberImpl) mucRoom).removePresencePacketExtensions(lowerHandExtension);
```

Comment:
Not really. The chatroom remove, removes that from the defaults we always add (intercept), sending is via the OpsSet will not adding multiple times, as it is checked and if it exists, will remove it first: https://github.com/jitsi/jitsi/blob/4b03fc6a2ec4b59112cf1c59cc6ba78c244b289f/modules/impl/protocol-jabber/src/main/java/net/java/sip/communicator/impl/protocol/jabber/ChatRoomJabberImpl.java#L2215  This logic was moved, it was used like that till now from the av moderation.

Reply:
https://github.com/jitsi/jitsi/blob/4b03fc6a2ec4b59112cf1c59cc6ba78c244b289f/modules/impl/protocol-jabber/src/main/java/net/java/sip/communicator/impl/protocol/jabber/ChatRoomJabberImpl.java#L1741

Reply:
So the `removePresencePacketExtensions(lowerHandExtension)` call, which only works with this one instance of `RaiseHandExtension`, is not necessary because it's going to be replaced anyway?

Reply:
Hum, I will double check, that and we can get rid of the removal then.

Reply:
Actually, there is nothing to check. We need to remove it as those are added via interceptor, so it will be the lower hand override the raise hand if not removed.